### ISSUE TITLE
training details

### ISSUE THREAD

**Issue by fcakyon** [obss/sahi#415]
Good afternoon. First of all congratulations on the success of your project. I had a question and I don't know if this is the right place. I have been trying to replicate the experiments regarding the dataset named Visdrone but I am having some problems. The parameters to perform the training are as follows:

SGD optimizer:
-Learning rate of 0.01
-Momentum of 0.9
-Weight decay of 0.0001
-Linear warmup of 500 iterations is used. 

How many epochs would you have to set to achieve the results presented in the paper?

On the other hand, when obtaining these results, which classes have been taken into account, all 90 COCO classes, or has it been limited to a series of them?

Thank you in advance

_Originally posted by @LawVP-SP in https://github.com/obss/sahi/issues/358#issuecomment-1072915268_
